multivariate-fine-mapping/mixture_prior.ipynb

```diff
@@ -437,21 +443,24 @@
     "parameter: ud_method = \"ed\"\n",
     "# A list of models where we only update the scales and not the matrices\n",
     "# A typical choice is to estimate scales only for canonical components\n",
-    "parameter: scale_only = []\n",
-    "\n",
+    "parameter: scale_only = \"canonical\"\n",
```

Comment:
@hsun3163 Please set default to `[]` and even if you want to put in default to `canonical` you need to use `["canonical"]` so SoS knows it is a list of multiple values. Otherwise your current implementation only allows one component to be fitted only for scale which is not general.